docs/modules/ROOT/pages/user-guide/command-mode.adoc

```diff
@@ -1,30 +1,251 @@
 = Command Mode Applications
 
-`camel-quarkus-core` brings the option to write https://quarkus.io/guides/command-mode-reference[Quarkus Command Mode Applications] with control about when the Camel runtime should start:
+Camel Quarkus is typically used for creating integration applications that run as long living processes.
+In this chapter, we are going to have a look at a slightly different use case:
+utilizing Camel Quarkus for writing programs that exit by themselves after performing some desired tasks.
+
+== Where can this be useful?
+
+The enterprise is full of scheduled batch processing. Say, some system exports some sort of reports daily at 4 a.m.
+Those need to be transformed (e.g. XML -> Excel) and transferred somewhere else (e.g. via mail, FTP, etc.)
+
+Camel is a perfect match for doing the transformation and the transfer.
+However, having a service running 24/7 for doing something useful just once a day would probably be an overkill.
+Combining a https://en.wikipedia.org/wiki/Cron[Cron job] (or a similar scheduling technology)
+with a command line utility may suit much better in situations like this.
+
+== How to write a command line tool with Camel Quarkus
+
+There are just one things where it would differ from a stock Camel Quarkus application:
```

Comment:
```suggestion
There is just one thing where it would differ from a stock Camel Quarkus application:
```

Reply:
+1, thanks!

Originally, there were two things and apparently I have not refactored the sentence properly.